SEBrowser/Scripts/TSX/Settings.tsx

```diff
@@ -29,10 +29,6 @@ import { Redux } from './global';
 import { FetchEventSearches } from './Components/EventSearch/EventSearchSlice';
 
 const searchSettingsOptions = [
-    {
-        Value: 'center',
-        Label: 'Center Date/Time and Window',
-    },

```

Comment:
Why are we dropping support for centered time window format?

Reply:
I believe that was the main update I was doing to this branch. Christoph said to do away with it, and just handle its conversion to the back end. (If it's still using that format in the back end)